### PR TITLE
[FEATURE] Support German (Germany)

### PR DESCRIPTION
**Pull request by XinyueZ**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <string name="plant_details_title">Details zur Pflanze</string>
+    <string name="navigation_drawer_header_content_description">App Navigation</string>
+    <string name="added_plant_to_garden">Pflanze wurde zum Garten hinzugefügt</string>
+    <string name="garden_empty">Ihr Garten ist unbepflanzt</string>
+    <string name="planted_date">%1$s gepflanzt in %2$s</string>
+    <string name="menu_item_share_plant">Teilen</string>
+    <string name="share_text_plant">Schau dir die %s Pflanze in der App "Sunflower" an</string>
+
+    <!-- String plurals and their related prefix / suffix strings -->
+    <!-- Item quantities for zero and one are not used in Chinese -->
```
Comment (tiembo) [inline app/src/main/res/values-de/strings.xml]: Can you remove this line?

Comment (XinyueZ) [reply app/src/main/res/values-de/strings.xml]: oha, sry, copy-paste mis.

File: app/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -28,7 +28,7 @@
     <string name="share_text_plant">%s という植物を「Sunflower」アプリで見てみてください</string>
 
     <!-- String plurals and their related prefix / suffix strings -->
-    <!-- Item quantities for zero and one are not used in Chinese -->
+    <!-- Item quantities for zero and one are not used in Japanese -->
```
Comment (tiembo) [inline app/src/main/res/values-ja/strings.xml]: Good catch - Fixed in #338

Comment (XinyueZ) [reply app/src/main/res/values-ja/strings.xml]: kkk

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    <string name="menu_item_share_plant">Teilen</string>
+    <string name="share_text_plant">Schau dir die %s Pflanze in der App "Sunflower" an</string>
+
+    <string name="watering_needs_prefix">Wasserbedarf:</string>
+    <plurals name="watering_needs_suffix">
+        <item quantity="one">jeden Tag</item>
+        <item quantity="other">alle %d Tage</item>
+    </plurals>
+
+    <string name="watering_next_prefix">Zuletzt bewässert am %s</string>
```
Comment (tiembo) [inline app/src/main/res/values-de/strings.xml]: Let's keep this comment in so it's easier for developers to understand:

```<!-- String plurals and their related prefix / suffix strings -->```

Comment (XinyueZ) [reply app/src/main/res/values-de/strings.xml]: fine

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<!--
+  ~ Copyright 2018 Google LLC
```
Comment (nic0lette) [inline app/src/main/res/values-de/strings.xml]: 2019 :)

Comment (XinyueZ) [reply app/src/main/res/values-de/strings.xml]: Well well

**Comment by tiembo** [comment]
Thanks @XinyueZ !

**Pull request merged**